src/components/ConnectionsList.tsx

```diff
@@ -0,0 +1,144 @@
+import React, { useEffect, useState } from "react";
+import { Button } from "antd";
+import ConnectionsListProfiles from "./ConnectionsListProfiles";
+import { useAppDispatch, useAppSelector } from "../redux/hooks";
+import { getProfile } from "../services/sdk";
```

Comment:
One thing I'd like to start doing for the sake of readability is using `* as sdk` for the sdk and also similar for our other internal APIs/services. With the goal of making it easier to navigate. Instead of having to scroll up to the top just to see where a method comes from, you can immediately tell. 

Examples would be things like:
`import * as sdk from "../services/sdk";`
`import * as wallet from "../services/wallet";`
`import * as session from "../services/session";`

And now our code, instead of having a random `getProfile` method that you now need to scroll up and figure out where it's from you can easily tell because it will become `sdk.getProfile`. Which makes it much more quickly apparent where it's from. It's slightly more verbose, but it's easier to follow which is our main goal.

Reply:
I like this idea a lot as far as readability. The only argument I have against it is that by importing with the `*` you are importing all exports from that file, which is unnecessary when you are only using, as in this case, one or two of the functions. Is it worth it? I'm not sure.